thunder/torch/__init__.py

```diff
@@ -1786,6 +1786,32 @@ def _reduction(
     return result
 
 
+@torchsymbol(torch.all, is_method=True, id="torch.all")
+def all_tensor(
```

Comment:
Would you add a comment explaining to future readers why the name is `all_tensor` instead of `all`? `all_tensor` is a great choice for the name, btw